oap-server/server-starter/src/main/resources/lal/mysql-slowsql.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  - name: mysql-slowsql
+    dsl: |
+      filter {
+        json{
+        }
+        extractor{
+          layer parsed.layer as String
+          service parsed.service as String
+          timestamp parsed.time as String
+          if (tag("LOG_KIND") == "SLOW_SQL") {
```

Comment:
I think `if (tag("LOG_KIND") == "SLOW_SQL")` `abort` should be added into other LAL rules. @kezhenxu94 Should we?

Reply:
@kezhenxu94 What do you recommend how we should set global rules to analyze logs?

Reply:
@yswdqz Have you checked, whether these logs are persistent in regular logs?

Reply:
@wu-sheng It will not be persistent.Because logs are persistent by sink filter.
If we configure the sink filter, it will be persistent.

Reply:
I mean there are other LAL scripts having sink filter. What do they not work for persistence? Such as `lal/default.yaml`

`envoy-als.yaml` should be safe as it has aborting conditions
```
        if (parsed?.response?.responseCode?.value as Integer < 400 && !parsed?.commonProperties?.responseFlags?.toString()?.trim()) {
          abort {}
        }
```

Reply:
I have not tested.
I am outside now, I will test them later.

Reply:
If `default` is not removed from 

https://github.com/apache/skywalking/blob/e6b5f32abe35dbb7e900afe0bcfba06b957fa904/oap-server/server-starter/src/main/resources/application.yml#L242-L242

they should be persisted.

Reply:
> I think `if (tag("LOG_KIND") == "SLOW_SQL")` `abort` should be added into other LAL rules. @kezhenxu94 Should we?

Yes I think so

Reply:
How about we merge this logic into default LAL script? I feel there is no benefit for maintaining and performance when keep them separated.

Reply:
I think you could edit default file. LAL is always pushing logs to all LAL scripts, that is by design.